features/gba-dev/Informatievragen-fields-gba.feature

```diff
@@ -0,0 +1,643 @@
+      | land.code         | 0000     |
+      | land.omschrijving | Onbekend |
+
+    Abstract Scenario: met fields gevraagd om <veld>
+      Gegeven de persoon met burgerservicenummer '000000309' heeft de volgende 'verblijfplaats' gegevens
+      | naam                                       | waarde           |
+      | functieAdres.code (10.10)                  | W                |
+      | datum aanvang adreshouding (10.30)         | 20150808         |
+      | gemeente van inschrijving (09.10)          | 0599             |
+      En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+      | naam                                       | waarde           |
+      | straatnaam (11.10)                         | Boterondiep      |
+      | naam openbare ruimte (11.15)               | Boterondiep      |
+      | huisnummer (11.20)                         | 31               |
+      | postcode (11.60)                           | 3077AW           |
+      | woonplaats (11.70)                         | Rotterdam        |
+      | gemeente_code                              | 0599             |
+      | identificatiecode verblijfplaats (11.80)   | 0599010000208579 |
+      | identificatiecode nummeraanduiding (11.90) | 0599200000219678 |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000309                       |
+      | fields              | <veld>                          |
+      Dan heeft de response een persoon met alleen de volgende 'verblijfplaats' gegevens
+      | naam                             | waarde               |
+      | straat                           | Boterondiep          |
+      | naamOpenbareRuimte               | <naamOpenbareRuimte> |
+      | huisnummer                       | 31                   |
+      | postcode                         | 3077AW               |
+      | woonplaats                       | Rotterdam            |
+      | adresseerbaarObjectIdentificatie | 0599010000208579     |
+      | nummeraanduidingIdentificatie    | 0599200000219678     |
+
+      Voorbeelden:
+      | veld                                | naamOpenbareRuimte |
+      | verblijfplaats.verblijfadres.straat | Boterondiep        |
+      | verblijfadres.straat                | Boterondiep        |
+      | straat                              | Boterondiep        |
+      | korteNaam                           |                    |
+      | postcode                            |                    |
+      | nummeraanduidingIdentificatie       |                    |
+      | locatiebeschrijving                 |                    |
+      | woonplaats                          |                    |
+      | verblijfadres.land                  |                    |
+      | regel1                              |                    |
+
```

Comment:
Volgens mij moet hier 3 keer een veblijfplaats met straat en openbare ruimtenaam geleverd worden en 7 keer een lege verblijfplaats. 

Misschien interpreteer ik zaken verkeerd, maar ik stop hier met reviewen. Veel scenario's falen terwijl volgens mij de implementatie door Kayo wel goed gedaan is. Ik vbraag me af waar de assertions op gebaseerd zijn. Bespreken bij de dagstart van vrijdag graag.